app/app.js

```diff
@@ -6,6 +6,9 @@ import { translate } from 'config/strings';
 import authenticated from 'auth';
 import storage from 'helpers/storage';
 import PetitionAPI from 'petition/api';
+import { CITY_PORTAL_PARAM, CITY_PORTAL_CLEAR_FILTER } from 'petition/api';
+import { retreiveCityPortalId } from 'petition/api';
```

Comment:
should already be fixed. i've pushed an additional commit. but github seems to be slow right now and does not show the latest update